utility/src/main/java/org/oppia/android/util/parser/html/CustomHtmlContentHandler.kt

```diff
@@ -208,6 +208,12 @@ class CustomHtmlContentHandler private constructor(
         CustomHtmlContentHandler(customTagHandlers, imageRetriever),
       ) as Spannable
     }
+
+    fun fromHtml(html: String): Spannable {
+      return HtmlCompat.fromHtml(
+        "<init-custom-handler/>$html", HtmlCompat.FROM_HTML_MODE_LEGACY,
```

Comment:
This should still be using CustomContentHandler.

It might actually be worth waiting until #3852 is merged since it's introducing a very similar function with tests which might make this a lot easier and avoid redundant work.

Reply:
Ok, then I will wait for #3852 to get merged and after that I will do required changes.